app/common/locales/en.json

```diff
@@ -980,6 +982,17 @@
         "finish_import" : "Finish import",
         "which_survey" : "which survey?"
     },
+    "data_export": {
+        "title": "Export data",
+        "description": "Export all of your data as a CSV or choose specific survey fields you want to export.",
```

Comment:
@Angamanga why is the description here different than the description in "export" key ?

Reply:
Its in different places, one for notifications and one for the export-view in settings...